domains/oslc-domains/pom.xml

```diff
@@ -38,23 +60,35 @@
                 <!-- End of user code
         -->
         <!-- General dependencies -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.7.36</version>
+            <scope>runtime</scope>
+        </dependency>
+        <!-- Servlet dependencies -->
         <dependency>
             <groupId>javax.servlet</groupId>
             <artifactId>javax.servlet-api</artifactId>
+            <version>3.1.0</version>
```

Comment:
The version is `<dependencyManagement>` versioned uniformly for all Lyo projects.